src/components/date-picker/date-picker.tsx

```diff
@@ -120,22 +128,25 @@ export class DatePicker implements LocalizedComponent {
    * Localized string for "previous month" (used for aria label)
    *
    * @default "Previous month"
```

Comment:
I think these no longer need defaults since it depends on the locale. I've been removing these entries in my PRs. @geospatialem @benelan WDYT?